magpie/register.py

```diff
@@ -739,6 +745,17 @@ def _parse_resource_path(permission_config_entry,   # type: PermissionConfigItem
         try:
             svc_name = service_info["service_name"]
             svc_type = service_info["service_type"]
+
+            # Prepare a list of types that fits with the list of resources
+            resource_type_list = resource_type_config.strip("/").split("/") if resource_type_config else [None]
```

Comment:
Yes, I can apply it to the `resource_path`. Just to be sure though, there's a slight difference between the 2. Currently it was just removing the first and last character if it's a '/', and with the `strip` it will remove every prefix and suffix '/'. Wasn't sure if it was important for the `resource_path`.